app/src/main/java/com/globant/practice/presentation/model/HomeViewState.java

```diff
@@ -0,0 +1,69 @@
+package com.globant.practice.presentation.model;
+
+import com.globant.practice.domain.model.User;
+import java.util.List;
+
+/**
+ * Manages the states of the view
+ * Created by jonathan.vargas on 18/04/2017.
+ */
+
+public class HomeViewState {
+    private boolean loading;
+    private List<User> users;
```

Comment:
Add a TODO comment here.
Ideally, the domain model must not be used in the View Model. In that case we have to recreate a similar User class for the view only, but seems this is a simple practice project we can skip it.